lib/utils.js

```diff
@@ -34,7 +35,9 @@ exports.addCheckedFunction = function (env) {
       return ''
     }
 
-    var storedValue = this.ctx.data[name]
+    // Use simple string keys or object notation
+    name = !name.match(/[.[]/g) ? `['${name}']` : name
+    var storedValue = getKeypath(this.ctx.data, name)
```

Comment:
Thanks @hannalaakso,

This is needed now we're using `keypather` to query the session data.

Without this line, all usage of `checked()` would need to follow `['']` notation which would break current prototypes. I didn't want to do that.

For example with the current kit it's like this:
```js
checked: checked("vehicle-features", "Heated seats")
```

Without this new line we'd need to change it to:
```js
checked: checked("['vehicle-features']", "Heated seats")
```

But, thankfully both are now supported for compatibility.

I've added a few examples in my latest push.